README.md

```diff
@@ -74,6 +82,27 @@ accept either the name or the ID of the branch, and you can use both (_i.e._,
 the `compare_branch` can use the branch name while the `base_branch` uses the
 branch ID or vice-versa).
 
+For the action to be able to create PR comments you must add the correct
```

Comment:
To enable the action to create pull request (PR) comments, you must configure the appropriate job permissions. To do this, add the following permissions to your job configuration: